app/assets/stylesheets/custom.css.scss

```diff
@@ -81,3 +81,66 @@ h1 {
     margin: auto;
     width:100%;
 }
+
+#flash_notice {
+  text-style: bold;
+  text-color: red;
+  opacity: .8;
+  text-align: center;
+  margin-right: 60%;
+  padding:4px;
+  border-radius: 100px;
+  color: cornsilk;
+  background-color: darkgrey;
+}
+
+#flash_errors{
+  text-style: bold;
+  text-color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Unknown property text-color